packages/Checkbox/tests/Checkbox.spec.js

```diff
@@ -0,0 +1,37 @@
+import { render } from "@testing-library/react";
+import React from "react";
+import Checkbox, { checkboxStates } from "../src/Checkbox";
+
+function renderComponent(props = {}) {
+  const defaultProps = {
+    a11yText: null,
+    checkedState: checkboxStates.CHECKED,
+    children: null,
+    isDisabled: false,
+    onChange: () => {},
+  };
+  return render(<Checkbox {...defaultProps} {...props} />);
+}
+
+describe("Checkbox", () => {
```

Comment:
Can we add a couple more tests?

One for checking a box, and other checkboxes are also selected.
Another for disabled